src/RefImplementations.cc

```diff
@@ -1180,6 +1179,42 @@ void transposeConvWeights(
   }
 }
 
+// Need these overloaded fuction to pass the compile since template with std::is_same<T, bfloat16> cannot work.
+// See https://stackoverflow.com/questions/50253286/using-stdis-same-why-my-function-still-cant-work-for-2-types
+void convert_to_float_ref(const bfloat16* src, float *dst){
```

Comment:
this set of overloads seems a bit odd, do we have more elegant solutions?

Reply:
I added two inline functions for conversion to make it can work with template.
[remove overloads](https://github.com/pytorch/FBGEMM/pull/1540/commits/5a06906ac5475496dc3b2eba4c383c4f97daa047)